sql/core/src/test/resources/sql-tests/results/udaf.sql.out

```diff
@@ -31,7 +31,7 @@ SELECT default.myDoubleAvg(int_col1, 3) as my_avg from t1
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-Invalid number of arguments for function spark_catalog.default.myDoubleAvg. Expected: 1; Found: 2; line 1 pos 7
+Invalid number of arguments for function spark_catalog.default.mydoubleavg. Expected: 1; Found: 2; line 1 pos 7
```

Comment:
This was a test problem before: we forgot to lower-case the function name when creating functions, and the testing in-memory catalog is case-preserving. This is inconsistent with table creation and should be fixed. Note that the production behavior (with Hive Metastore) is not changed: Hive Metastore is not case-preserving.